oauth2client/contrib/appengine.py

```diff
@@ -62,6 +61,21 @@
 
 XSRF_MEMCACHE_ID = 'xsrf_secret_key'
 
+if _appengine_ndb is None:
+    CredentialsNDBModel = None
+    CredentialsNDBProperty = None
+    FlowNDBProperty = None
+    _NDB_KEY = None
+    _NDB_MODEL = None
+    SiteXsrfSecretKeyNDB = None
+else:
+    CredentialsNDBModel = _appengine_ndb.CredentialsNDBModel
+    CredentialsNDBProperty = _appengine_ndb.CredentialsNDBProperty
+    FlowNDBProperty = _appengine_ndb.FlowNDBProperty
+    _NDB_KEY = _appengine_ndb.NDB_KEY
+    _NDB_MODEL = _appengine_ndb.NDB_MODEL
```

Comment:
I should have caught that in review; thanks for letting me know.

If we were lint-compliant we would have gotten a "suspicious access of private element" warning, wouldn't we have? In a perfect world. :-P